openfl/utilities/utils.py

```diff
@@ -236,3 +236,27 @@ def merge_configs(
             settings.set(key, operation(value))
     settings.validators.validate()
     return settings
+
+
+def change_tags(tags, add_field=None, remove_field=None) -> Tuple[str]:
+    """Change tensor tags to add or remove fields and change tensor type.
+
+    Args:
+        tags(list or tuple): tensor tags.
+        add_field(str): add a new tensor tag field.
+        remove_field(str): remove a tensor tag field.
+    """
+    if add_field:
+        if type(tags) == str:
```

Comment:
Why do we expect `str`? Could the conversion be executed outside this method, so we operate only with tuples/lists here?

Reply:
I think that `change_tags` should operate only with `tuple`

Reply:
Done